hack/fuse-demo/run_coord_shell.sh

```diff
@@ -15,14 +15,16 @@ while getopts s opt; do
 done
 (( OPTIND > 1 )) && shift $(( OPTIND - 1 ))
 
+typeset pod_name
 
-pod_name=$(kubectl get pods -l app=datamon-coord-demo | grep Running | sed 's/ .*//')
+print -- "waiting on pod start"
 
-if [[ -z $pod_name ]]; then
-	echo 'coord demo pod not found' 1>&2
-	exit 1
-fi
+while [[ -z $pod_name ]]; do
+    sleep 1
+    pod_name=$(kubectl get pods -l app=datamon-coord-demo | \
```

Comment:
Just saying here... From my experience, having a pod in a running state does'nt mean its service is ready. Don't we have a ready probe that could be queried here? Like:
```
until printf "." && curl -fs --insecure --connect-timeout 2 https://localhost:8443/auth >/dev/null ; do
  sleep 2;
done
```
Or use `nc` instead of `curl`

Reply:
understood.

this is more stuff that "just works."  moreover, for the purposes of starting a shell within the pod, the intent of this script, having the pod in a Running state could be precisely what's necessary?

conversely, it should be understood that the "service layer" is the cross-container coordination mechanism, which is currently implemented as files that could be stat'ed.  and yes, there could be an additional event(s) added to `wrap_application.sh` for health checks, etc.  this is another item i'm willing to file under technical debt and cleanup as part of this merge request or a later one, depending on how much time is left after the requirements management and functionality is pieces are settled.